core/src/bank_forks.rs

```diff
@@ -421,7 +423,7 @@ impl BankForks {
             working_bank,
             root,
             snapshot_config: None,
-            last_snapshot: *names.last().unwrap(),
+            slots_since_snapshot: vec![slot_paths.last().unwrap().slot],
```

Comment:
should this just be vec![] . because the last slot was included . in the last snapshot